### PR TITLE
support ie8

### PR DESCRIPTION
**Pull request by mjneil**
addresses #9

### CODE REVIEW

File: src/quality-level.js [hunk 1]
```diff
@@ -25,13 +28,26 @@ export default class QualityLevel {
...
+    }
+
+    level.id = representation.id;
+    level.label = level.id;
+    level.width = representation.width;
+    level.height = representation.height;
+    level.bitrate = representation.bandwidth;
+    level.enabled_ = representation.enabled;
+
+    return level;
```
Comment (gkatsev) [inline src/quality-level.js]: just doing this makes the `get` syntax below work fine?

Comment (mjneil) [reply src/quality-level.js]: Turns out no. I had my IE emulation settings wrong when testing the behavior

Comment (gkatsev) [reply src/quality-level.js]: IE emulation settings? Make sure to test in IE8 directly. compatibility modes in newer IEs aren't sufficient.

**Comment by forbesjo** [comment]
Looks good to me

**Pull request merged**